Sources/HTTP/HTTPVersion.swift

```diff
@@ -29,12 +29,44 @@ extension HTTPVersion : Hashable {
     public static func == (lhs: HTTPVersion, rhs: HTTPVersion) -> Bool {
         return lhs.hashValue == rhs.hashValue
     }
-
```

Comment:
I'll update to remove the potential hash collision and replace the implementation on `==`